R/clean_spelling_mistakes.R

```diff
@@ -0,0 +1,98 @@
+#' Clean spelling mistakes detected with approximate string matching and
+#' replaced with `wordlist`
+#'
+#' @details
+#' When used interactively (see [interactive()]) the user is presented a menu
+#' to ensure that the words detected using approximate string matching are not
+#' false positives and the user can decided whether to proceed with the
+#' spelling corrections. In non-interactive sessions all spelling mistakes
+#' detected with approximate string matching are replaced by the `wordlist`
+#' word.
+#'
+#' @inheritParams clean_data
+#' @param wordlist A `character` vector or list of words to match to cells in
+#' `data`.
+#' @inheritParams base::agrep
```

Comment:
Should these be passed as `...` for future-proofing the potential addition of new arguments to `agrep()`?

Reply:
Resolved in 142d1251b3b718d59f4e54f8279fb2adab12c20a.